src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/_params.py

```diff
@@ -8,28 +8,72 @@
+
+register_cli_argument('batch pool resize', 'if_modified_since', help='Specify this header to perform the operation only if the resource has been modified since the specified date/time.', type=datetime_format, arg_group='Pre-condition')
+register_cli_argument('batch pool resize', 'if_unmodified_since', help='Specify this header to perform the operation only if the resource has not been modified since the specified date/time.', type=datetime_format, arg_group='Pre-condition')
+register_cli_argument('batch pool resize', 'if_match', help='An ETag is specified. Specify this header to perform the operation only if the resource\'s ETag is an exact match as specified', arg_group='Pre-condition')
+register_cli_argument('batch pool resize', 'if_none_match', help='An ETag is specified. Specify this header to perform the operation only if the resource\'s ETag does not match the specified ETag.', arg_group='Pre-condition')
+register_cli_argument('batch pool resize', 'pool_id', help='The ID of the pool.')
+register_cli_argument('batch pool resize', 'abort', action='store_true', help='Stop the pool resize operation.', validator=validate_pool_resize_parameters)
+
+register_cli_argument('batch pool reset', 'json_file', type=file_type, help='The file containing PoolUpdatePropertiesParameter object in JSON format, if this parameter is specified, all other parameters are ignored.', validator=validate_json_file, completer=FilesCompleter())
+register_cli_argument('batch pool reset', 'pool_id', help='The ID of the pool to be updated.')
+register_cli_argument('batch pool reset', 'application_package_references', nargs='+', type=application_package_reference_format)
+register_cli_argument('batch pool reset', 'certificate_references', nargs='+', type=certificate_reference_format)
+register_cli_argument('batch pool reset', 'metadata', nargs='+', type=metadata_item_format)
+
+register_cli_argument('batch job list', 'filter', help=' An OData $filter clause.', arg_group='OData')
+register_cli_argument('batch job list', 'select', help=' An OData $select clause.', arg_group='OData')
+register_cli_argument('batch job list', 'expand', help=' An OData $expand clause.', arg_group='OData')
+register_cli_argument('batch job list', 'job_schedule_id', help='The ID of the job schedule from which you want to get a list of jobs.')
+
+register_cli_argument('batch certificate', 'thumbprint', help='The certificate thumbprint.')
+register_cli_argument('batch certificate', 'thumbprint_algorithm', help='The certificate thumbprint algorithm.')
+register_cli_argument('batch certificate', 'password', help='The password to access the certificate\'s private key.')
+register_cli_argument('batch certificate', 'cert_file', type=file_type, help='The certificate file: cer file or pfx file.', validator=validate_cert_file, completer=FilesCompleter())
```

Comment:
You should also specify `type=file_type` here and import file_type from `azure.cli.core.commands.parameters`. This ensures the leading ~ is expanded correctly.